include/twin.h

```diff
@@ -692,6 +709,19 @@ void twin_icon_draw(twin_pixmap_t *pixmap,
 
 twin_pixmap_t *twin_pixmap_from_file(const char *path, twin_format_t fmt);
 
+/*
+ * image-gif.c
```

Comment:
Refine this comment for dedicated image manipulation interface file.

You can follow the idea how [GdkPixbuf : PixbufAnimation](https://docs.gtk.org/gdk-pixbuf/index.html) deals with the iterator.

In addition, you should give more comments for each function introduced for the animation.